src/models/User.ts

```diff
@@ -0,0 +1,5 @@
+export default class User {
+    constructor() {
+        console.log('User module');
```

Comment:
it's better to use logging abstraction like https://www.npmjs.com/package/winston

Reply:
I guess for this task we need this logs just to check that server is running, not for real logging